test/native.auth.test.ts

```diff
@@ -255,6 +256,105 @@ describe("Native Auth", () => {
       }));
     });
 
+    it('General wildcard is accepted', async () => {
+      const server = new NativeAuthServer({
+        ...defaultConfig,
+        acceptedOrigins: ['*'],
+      });
+
+      onSpecificBlockTimestampGet(mock).reply(200, BLOCK_TIMESTAMP);
+      onLatestBlockTimestampGet(mock).reply(200, [{ timestamp: BLOCK_TIMESTAMP }]);
+
+      const result = await server.validate(ACCESS_TOKEN);
+      expect(result).toStrictEqual(new NativeAuthResult({
+        address: ADDRESS,
+        signerAddress: ADDRESS,
+        issued: BLOCK_TIMESTAMP,
+        expires: BLOCK_TIMESTAMP + TTL,
+        origin: ORIGIN,
+      }));
+    });
+
+    it('Wildcard origin is accepted', async () => {
+      const server = new NativeAuthServer({
+        ...defaultConfig,
+        acceptedOrigins: ['*.multiversx.com'],
+      });
+
+      onSpecificBlockTimestampGet(mock).reply(200, BLOCK_TIMESTAMP);
+      onLatestBlockTimestampGet(mock).reply(200, [{ timestamp: BLOCK_TIMESTAMP }]);
+
+      const result = await server.validate(ACCESS_TOKEN);
+      expect(result).toStrictEqual(new NativeAuthResult({
+        address: ADDRESS,
+        signerAddress: ADDRESS,
+        issued: BLOCK_TIMESTAMP,
+        expires: BLOCK_TIMESTAMP + TTL,
+        origin: ORIGIN,
+      }));
+    });
+
+    it('Wildcard validation two wildcards not accepted', () => {
+      const config: NativeAuthServerConfig = {
+        ...defaultConfig,
+        acceptedOrigins: ['*.multiversx*.com'],
+      };
+
+      expect(() => new NativeAuthServer(config)).toThrow(NativeAuthInvalidWildcardOriginError);
+    });
+
+    it('Wildcard validation protocol not accepted', () => {
+      const config: NativeAuthServerConfig = {
+        ...defaultConfig,
+        acceptedOrigins: ['www.*.multiversx.com'],
+      };
+
+      expect(() => new NativeAuthServer(config)).toThrow(NativeAuthInvalidWildcardOriginError);
+    });
+
+    it('Wildcard origin is not accepted', async () => {
+      const server = new NativeAuthServer({
+        ...defaultConfig,
+        acceptedOrigins: ['*.elrond.com'],
```

Comment:
in my opinion this should be a valid input

Reply:
it is invalid because the token is sent for validation with origin 'https://api.multiversx.com' which does not match the requirements

Reply:
(all tokens sent for validation contain https://api.multiversx.com)